deployment/helm/resource-management-policies/balloons/templates/configmap.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nri-resource-policy-config.default
```

Also here, the name needs to be distinctive

```suggestion
  name: nri-resource-policy-balloons-config.default
```

plus the labels...